pkg/scripts/os.go

```diff
@@ -231,28 +231,26 @@ sudo yum install -y \
 	{{ amznYumDocker .KUBERNETES_VERSION }}
 sudo yum versionlock add docker
 
-sudo systemctl daemon-reload
-sudo systemctl enable --now docker
+sudo mkdir -p /opt/bin /etc/kubernetes/pki /etc/kubernetes/manifests
 
-sudo mkdir -p /tmp/k8s-binaries /etc/kubernetes/pki /etc/kubernetes/manifests
-
-sudo rm -rf /tmp/k8s-binaries
-sudo mkdir /tmp/k8s-binaries
+rm -rf /tmp/k8s-binaries
+mkdir -p /tmp/k8s-binaries
 cd /tmp/k8s-binaries
 
 {{- if .CNI_URL }}
-sudo mkdir -p /usr/cni/bin
-curl -L "{{ .CNI_URL }}" | sudo tar -C /usr/cni/bin -xz
+sudo mkdir -p /opt/cni/bin
+curl -L "{{ .CNI_URL }}" | sudo tar -C /opt/cni/bin -xz
 {{- end }}
 
 {{- if .NODE_BINARIES_URL }}
-sudo curl -L --output /tmp/k8s-binaries/node.tar.gz {{ .NODE_BINARIES_URL }}
-sudo tar xvf node.tar.gz
+curl -L --output /tmp/k8s-binaries/node.tar.gz {{ .NODE_BINARIES_URL }}
+tar xvf node.tar.gz
 {{- end }}
 
 {{- if and .KUBELET .NODE_BINARIES_URL }}
-sudo install --owner=0 --group=0 --mode=0755 /tmp/k8s-binaries/kubernetes/node/bin/kubelet /usr/bin/kubelet
-sudo rm /tmp/k8s-binaries/kubernetes/node/bin/kubelet
+sudo install --owner=0 --group=0 --mode=0755 /tmp/k8s-binaries/kubernetes/node/bin/kubelet /opt/bin/kubelet
+sudo ln -sf /opt/bin/kubelet /usr/bin/
```

Comment:
Do we even need to link kubelet if in systemd unit file it's used like `ExecStart=/opt/bin/kubelet`?

Reply:
The reason for doing this is that Kubeadm requires Kubelet to be on PATH. Otherwise, the preflight checks would fail with the following error:

```
[preflight] Some fatal errors occurred:
	[ERROR KubeletVersion]: couldn't get kubelet version: cannot execute 'kubelet --version': executable file not found in $PATH
```